pkg/util/storage/manager.go

```diff
@@ -82,3 +86,41 @@ func (m *manager) BlobService(ctx context.Context, resourceGroup, account string
 
 	return &blobcli, nil
 }
+
+func (m *manager) CreateBlobContainer(ctx context.Context, resourceGroup string, account string, container string) error {
```

Comment:
Do we need to create a storage account in our e2e environment (RP health subscription)? I believe they were only created in shared dev, canary, and INT environments.

Reply:
Yes good point, we will need the storage account in e2e too. Currently updating the secrets for e2e to add `STORAGE_ACCOUNT_DOMAIN` to form the correct storage account name.